### PR TITLE
Update account settings

### PR DESCRIPTION
**Pull request by trowland10**
changed sql queries and added an e-mail check, and removed preferences from the dropdown menu. Deleted preferences.handlebars file

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -324,19 +322,26 @@ router.post('/updateaccount', common.isAuthenticated, (req, res, next) => {
...
-    sql.pool.query('UPDATE Users SET firstName=?, lastName=?, email=?, address=?, city=?, state=?, postalCode=?, country=?, worldwide=?, aboutMe=? WHERE id=?',
-    [data.firstName, data.lastName, data.email, data.address, data.city, data.state, data.postalCode, data.country, data.worldwide, data.aboutMe, data.userId],
+    sql.pool.query('SELECT id, email FROM Users WHERE email=?', [data.email],
     (err, results) => {
-        if (err) {
-            req.flash('error', err)
-            res.render('auth/updateaccount', data)
+        if (results.length != 0 && results[0].id != req.session.userId) {
+            req.flash("error", "This email is already associated with another account")
+            res.reload();  //this might need to be changed
```
Comment (mkoeplin) [inline routes/index.js]: If the email already exists you won't want to do a reload as you'll clear out all the data they already entered. You should be using res.render here and passing back the data object you have. Typically in this case I would se the data.email to blank before sending back so it clears that field but leaves the rest. I'd also put back the title in the data so you're returning and still have the tab id or page title which should be using that as well.

**Review by mkoeplin — CHANGES_REQUESTED** [review]
Let's fix the return if the email already exists to send back the data they entered so they don't have to re-enter any other changes. See my notes on the file. We should also always be sending in a data.title when we return to a page.

File: routes/index.js (hunk 1 above)
Comment (trowland10) [reply routes/index.js]: my new commit reflects these changes.

**Review by mkoeplin — APPROVED** [review]
Thanks for making that change!

**Review by halekyl — APPROVED** [review]
I don't see any issues, looks good to go.

**Pull request merged**